schedule/yast/maintenance/yast_libyui.yaml

```diff
@@ -0,0 +1,11 @@
+name: yast_libyui
+vars:
+  DESKTOP: textmode
+  YUI_REST_API: 1
+  YUI_HTTP_PORT: 39125
+  YUI_HTTP_REMOTE: 1
+  YUI_REUSE_PORT: 1
+  
+schedule:
+  - installation/bootloader
+  - installation/setup_libyui
```

Comment:
Why do we need this yaml file?

Reply:
this was just for testing, please, remove

Reply:
got it.